packages/salesforcedx-vscode-core/src/commands/templates/forceLightningAppCreate.ts

```diff
@@ -34,6 +37,31 @@ import {
   AURA_TYPE
 } from './metadataTypeConstants';
 
+export class LibraryForceLightningAppCreateExecutor extends LibraryBaseTemplateCommand<
+  DirFileNameSelection
```

Comment:
How come a new base class is being used instead of BaseTemplateCommand?

Reply:
This is a follow up of #2428 where [`LibraryBaseTemplateCommand`](https://github.com/forcedotcom/salesforcedx-vscode/pull/2428/files#diff-81cf0fdec768a9dede6cce86cb33ffc6) is defined. It's designed to substitute `BaseTemplateCommand` - so we can remove `BaseTemplateCommand` in the future. Please take a look at the original PR.